src/libpriv/rpmostree-core.c

```diff
@@ -2058,7 +2058,8 @@ on_async_import_done (GObject                    *obj,
 
   g_assert_cmpint (self->n_async_pkgs_imported, <, self->pkgs_to_import->len);
   self->n_async_pkgs_imported++;
-  dnf_state_assert_done (self->async_dnfstate);
+  rpmostree_output_progress_n_items ("Importing", self->n_async_pkgs_imported,
+                                     self->pkgs_to_import->len);
```

Comment:
To be consistent, let's use this in the relabel case too? And then we can drop `async_dnfstate` I think.

Reply:
Yep done :arrow_down: